tests/vcs/git/GitCommitBearTest.py

```diff
@@ -125,6 +127,45 @@ def test_empty_message(self):
                          [])
         self.assert_no_msgs()
 
+    def test_revert_checks(self):
+        self.run_git_command('commit', stdin='Test commit')
```

Comment:
Please use ```self.run_git_command``` for committing only after doing a ```git add```.
Otherwise this will fail due to empty diffs and as a result no commit would be made. 
And a ```git rev-parse HEAD``` on a repository with no commit will result in failures.

Consider using ```self.git_commit``` instead as it allows empty commits and can be done even without doing a ```git add```

Rather if you want to use ```run_git_command```, use it to generate a revert commit 
For e.g ```self.run_git_command('revert', 'head')```